llvm/test/CodeGen/DirectX/ShaderFlags/double-extensions.ll

```diff
@@ -3,7 +3,7 @@
 
 target triple = "dxil-pc-shadermodel6.7-library"
 
-; CHECK: ; Shader Flags Value: 0x00000021
+; CHECK: ; Shader Flags Value: 0x00000044
```

Comment:
Do the extra flags get pretty-printed below? If not, should they? Should we check them?

Reply:
The following CHECK lines are the pretty-printed.

Reply:
There are new flags you've added which don't have tests verifying the pretty printing.

Reply:
We don't have all the required dxil operation or compiler option added for the full list of DxilModuleFlags. If a DxilModuleFlag is not set, it will not be printed.
Test will be added when we really set these flags.

Reply:
For double flag, which already supported.
The print check is in the following CHECK lines as
; CHECK-NEXT: ;       Double-precision floating point
; CHECK-NEXT: ;       Double-precision extensions for 11.1

We don't support other flags right now.
We don't have dxil intrinsic like waveOps.

Reply:
Repeating the same thing back to me doesn't help me understand the issue here.

I went and looked at the code, and I think we have a separate issue we need to resolve, but that can be separate. We should be able to dump the flags from the computed value that is stored in the module, but today the printer only operates on the analysis result. We can address that separately.